scale/job/models.py

```diff
@@ -391,7 +391,7 @@ def get_details(self, job_id):
                 job.execution = None
         else:
             job.execution = None
-
+        job.configuration = job.job_type.configuration
```

Comment:
We should use the get_configuration method to scrub secrets or something similar